src/pkg/backup/details/testdata/in_deets_test.go

```diff
@@ -234,7 +234,100 @@ func (suite *LocSetUnitSuite) TestRename() {
 	}
 }
 
-func (suite *LocSetUnitSuite) TestMove() {
+func (suite *LocSetUnitSuite) TestItem() {
+	t := suite.T()
+	b4 := "bar/lr_4"
+
+	makeSet := func() *locSet {
+		ls := newLocSet()
+
+		ls.AddItem(l1, i1)
+		ls.AddItem(l1, i2)
+		ls.AddLocation(l13)
+		ls.AddItem(l14, i3)
+		ls.AddItem(l14, i4)
+		ls.AddItem(b4, "fnord")
+
+		return ls
+	}
+
+	ts := makeSet()
+	assert.ElementsMatch(t, []string{l1, l13, l14, b4}, maps.Keys(ts.Locations))
+	assert.ElementsMatch(t, []string{i1, i2}, maps.Keys(ts.Locations[l1]))
+	assert.Empty(t, maps.Keys(ts.Locations[l13]))
+	assert.ElementsMatch(t, []string{i3, i4}, maps.Keys(ts.Locations[l14]))
+	assert.ElementsMatch(t, []string{"fnord"}, maps.Keys(ts.Locations[b4]))
```

Comment:
these should maybe be `require`? That would ensure the other tests don't fail due to failures when setting up the data

Reply:
eh, failure is failure, and it's not a very complicated test.